index.html

```diff
@@ -272,5 +272,73 @@ <h2>About me</h2>
         </div>
       </div>
     </main>
+    <footer class="footer">
+      <form
+        action="https://formspree.io/f/contact-form"
+        method="post"
+        class="contact-form"
+      >
```

Comment:
- [ ] Good job :100: :+1: with your `Formspree`. So, I noticed that the user can't send a message after completing all fields and clicking the `Get in touch` button. I think you should create an **account** and you'll have a `unique` key that you will use in: ` action="https://formspree.io/f/form_id"`.